### PR TITLE
Fixing some tiny glitches

### PR DESCRIPTION
**Pull request by jakobsack**
In this PR I fix some glitches I found. Some of them might be a matter of taste.

### CODE REVIEW

File: lib/Client.php [hunk 1]
```diff
@@ -551,7 +552,6 @@ protected function sendAsyncInternal(RequestInterface $request, callable $succes
             $curl,
             $this->createCurlSettingsArray($request)
         );
-        curl_multi_add_handle($this->curlMultiHandle, $curl);
```
Comment (evert) [inline lib/Client.php]: Should this have been deleted?

Comment (jakobsack) [inline lib/Client.php]: Oups, I intented to delete this line but I double checked and it is actually needed by poll(). Seems like I got lost in the code flow. I'll change the PR soon.

**Comment by jakobsack** [comment]
@evert I fixed the issue you pointed out. If you are happy with the PR you can merge it, I don't have anything to add.

**Comment by evert** [comment]
Thank you!

**Pull request merged**